src/absent-announce.ts

```diff
@@ -153,11 +154,18 @@ const createMessage = (
   const creatorEmail = event.getCreators()[0];
   const name = convertEmailToName(creatorEmail);
 
-  return event.isAllDayEvent()
-    ? `【全休】 ${name}さん ${eventStartDate}〜${eventEndDate} 終日\n`
-    : `【半休】 ${name}さん ${eventStartDate} ${eventStartTime}〜${eventEndTime}\n`;
+  return isWholeDayOff(event)
+    ? `${name}さん ${eventStartDate}〜${eventEndDate} 全休\n`
+    : `${name}さん ${eventStartDate} ${eventStartTime}〜${eventEndTime}\n`;
 };
 
+const isWholeDayOff = (
+  event: GoogleAppsScript.Calendar.CalendarEvent
+): boolean => {
+  const eventStart = new Date(event.getStartTime().getTime());
+  const eventEnd = new Date(event.getEndTime().getTime());
+  return differenceInHours(eventEnd, eventStart) >= 6;
```

Comment:
6hって数字を後から見返したら「通常勤務時間は8hなのになぜ？」となりそなので、一応コメント残しておくほうが良さそう

```suggestion
  // 6時間以上を全休と定義
  return differenceInHours(eventEnd, eventStart) >= 6;
```

Reply:
対応しました　e3f106c0998984be3f90973876d0a676fcb63a60